uportal-war/src/main/webapp/WEB-INF/jsp/BackgroundPreference/viewBackgroundPreference.jsp

```diff
@@ -139,8 +139,8 @@
         </a>
         <c:forEach var="image" items="${thumbnailImages}" varStatus="status">
             <a href="#">
-                <img src="${image}" />
-                <span class="caption">Background ${status.index + 1}</span>
+                <img src="${image}" alt=${imageCaptions[status.index]}/>
+                <span class="caption">${imageCaptions[status.index]}</span>
```

Comment:
Looks to me like `imageCaptions` defaults to `null` in `RoleBasedBackgroundSetSelectionStrategy` line 129. Will this JSP fail gracefully if the image captions portlet-preference hasn't been set?